core/src/main/java/hudson/PluginManager.java

```diff
@@ -78,84 +60,44 @@
 import org.apache.commons.logging.LogFactory;
 import org.jenkinsci.Symbol;
 import org.jenkinsci.bytecode.Transformer;
-import org.jvnet.hudson.reactor.Executable;
-import org.jvnet.hudson.reactor.Reactor;
-import org.jvnet.hudson.reactor.ReactorException;
-import org.jvnet.hudson.reactor.TaskBuilder;
-import org.jvnet.hudson.reactor.TaskGraphBuilder;
+import org.jvnet.hudson.reactor.*;
+import org.kohsuke.accmod.Restricted;
 import org.kohsuke.accmod.restrictions.DoNotUse;
-import org.kohsuke.stapler.HttpRedirect;
-import org.kohsuke.stapler.HttpResponse;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
+import org.kohsuke.stapler.*;
```

Comment:
Is it allowed? I don't know for sure

Reply:
It is not :) (except for assert/matchers in test)

Reply:
I had this doubt when I saw the IDE changed it